connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorValidationIntegrationTest.java

```diff
@@ -343,7 +346,7 @@ public void testConnectorHasConverterWithNoSuitableConstructor() throws Interrup
     @Test
     public void testConnectorHasConverterThatThrowsExceptionOnInstantiation() throws InterruptedException {
         Map<String, String> config = defaultSinkConnectorProps();
-        config.put(KEY_CONVERTER_CLASS_CONFIG, TestConverterWithConstructorThatThrowsException.class.getName());
+        config.put(KEY_CONVERTER_CLASS_CONFIG, TestPlugins.TestPlugin.ALWAYS_THROW_EXCEPTION.className());
```

Comment:
We could probably use `BAD_PACKAGING_DEFAULT_CONSTRUCTOR_THROWS_CONNECTOR` and in `testConnectorHasHeaderConverterThatThrowsExceptionOnInstantiation` as well?

Reply:
At the beginning I used `BAD_PACKAGING_DEFAULT_CONSTRUCTOR_THROWS_CONNECTOR` in the case, but I thought `ALWAYS_THROW_EXCEPTION` has stronger sematics, that's why I changed my mind.

But now I think about the case again, I think `BAD_PACKAGING_DEFAULT_CONSTRUCTOR_THROWS_CONNECTOR ` has fine-grain sematics which is more suitable for the case.

So I will change it. Thanks so much for pointing out that.